### ISSUE TITLE
Best Practices: Drupal 8 custom code

### ISSUE THREAD

**Issue by joesb** [wunderkraut/WunderWay#152]
Custom code:

- where do you put it?
- how do you split it up?
- how do you keep it clean from contrib, etc etc.?
- gotchas?
- working with dependency injection
- how's it different from D7?

Worth blogging on?

Depends on #149 
Relates to #151
